queries.sql

```diff
@@ -117,4 +117,55 @@ SELECT a.name, a.escape_attemps from animals a
 SELECT o.full_name, count(a.owner_id) as owns from owners o
     JOIN animals a ON o.id = a.owner_id
         GROUP by o.full_name
-            HAVING MAX(a.owner_id) = 3;
\ No newline at end of file
+            HAVING MAX(a.owner_id) = 3;
+
+/* ------------------------------------------------------------- */
+
+-- SELECT max(vi.date_of_visit), vi.animal_id from visits vi
+--     GROUP BY vi.animal_id, vi.vet_id
+--     HAVING vi.vet_id = (SELECT id from vets 
+--                         WHERE name = 'William Tatcher');
+
+-- SELECT vi.animal_id from visits vi
+--     GROUP BY vi.animal_id, vi.vet_id
+--     HAVING vi.vet_id = (SELECT id from vets 
+--                         WHERE name = 'William Tatcher');
+
```

Comment:
- [ ] Great job so far however, we missing to write queries to answer the following: Who was the last animal seen by William Tatcher? Kindly add this 👍🏻